components/atlas-uvs/README.md

```diff
@@ -26,8 +27,8 @@ Install and use by directly including the [browser files](dist):
 ```html
 <head>
   <title>My A-Frame Scene</title>
-  <script src="https://aframe.io/releases/0.9.0/aframe.min.js"></script>
-  <script src="https://unpkg.com/aframe-atlas-uvs-component@2.1.0/dist/aframe-atlas-uvs-component.min.js"></script>
+  <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+  <script src="https://unpkg.com/aframe-atlas-uvs-component@3.0.0/dist/aframe-atlas-uvs-component.min.js"></script>
```

Comment:
Yeah you were right. We can bump the version in the package.json. In A-Frame we update README and examples automatically when cutting a release. We don't have this automated here.